static/js/portico/help.js

```diff
@@ -113,10 +113,16 @@ function render_code_sections() {
 
         history.pushState({}, "", path);
 
+        var hash = window.location.hash;
         if (html_map[path]) {
             $(".markdown .content").html(html_map[path]);
             Ps.update(container);
             render_code_sections();
+            if (hash !== '') {
+                container.scrollTop = $(hash).position().top - $('.markdown .content').position().top;
+            } else {
+                container.scrollTop = 0;
+            }
```

Comment:
Can you put this block into a function so we don't have to copy it twice?

Reply:
Can I refactor from line number 131-138?